.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # Automatically generated CODEOWNERS
 # Regenerate with `make update-codeowners`
-draft-ecahc-moderation.md lars@eggert.org jari.arkko@ericsson.com housley@vigilsec.com brian.e.carpenter@gmail.com
+draft-ietf-modpod-group-processes.md lars@eggert.org lear@lear.ch
```

Comment:
@elear what is the email address you use on GitHub? (It's not this one.)

Reply:
Fine with me.

Reply:
Thanks Russ. (Chairs assigned editors.)